crates/aptos/src/move_tool/bytecode.rs

```diff
@@ -201,6 +219,66 @@ impl BytecodeCommand {
         })
     }
 
+    fn print_metadata(&self, bytecode_path: &Path) -> Result<String, CliError> {
+        let bytecode_bytes = read_from_file(bytecode_path)?;
+
+        let module: CompiledModule;
```

Comment:
These two variable declarations can instead be moved inside the corresponding if-else bodies and be initialized.

Reply:
done